src/bin/nile-rs/commands/clean.rs

```diff
@@ -0,0 +1,63 @@
+use anyhow::{Context, Result};
+use scarb::core::Config;
+use scarb::ops;
+use std::fs;
+use std::path::PathBuf;
+use std::thread;
+
+use super::CliCommand;
+use async_trait::async_trait;
+use clap::Parser;
+use nile_rs::config::Config as NileConfig;
+
+#[derive(Parser, Debug)]
+pub struct Clean {
+    #[clap(
+        help = "Scarb manifest path",
+        long,
+        short,
+        default_value = "./Scarb.toml"
+    )]
+    pub manifest_path: String,
+}
+
+#[async_trait]
+impl CliCommand for Clean {
+    type Output = ();
+
+    // Remove artifacts from workspace
+    async fn run(&self) -> Result<Self::Output> {
+        // Remove the deployments directory
+        let nile_config = NileConfig::get()?;
+        let deployments_dir = nile_config.deployments_dir;
+        if PathBuf::from(&deployments_dir).exists() {
+            fs::remove_dir_all(deployments_dir).unwrap();
```

Comment:
Following up on the previous comment, I think is better not to remove accounts from this command, or at least add a specific flag for that action.

Reply:
Will update this 👍